api/src/migrations/1575734358119-add-label-join-tables.ts

```
@@ -156,7 +156,7 @@ export class addLabelJoinTables1575734358119 implements MigrationInterface {
         await queryRunner.query('DROP TABLE IF EXISTS "label_terms_term"');
         break;
       default:
-        console.log('Unknown DB type');
+        throw new Error('Unknown DB type: ' + config.db.default.type);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)